src/Npgsql/Internal/Converters/RangeConverter.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Npgsql.Internal.Converters;
 
-sealed class RangeConverter<T> : PgStreamingConverter<NpgsqlRange<T>>
+public class RangeConverter<T> : PgStreamingConverter<NpgsqlRange<T>>
```

Comment:
Should we just add AddRangeType/AddStructRangeType to the mapping collection? Do we want to support manually constructed plugin resolvers?

Reply:
I made this public also because the specialized NodaTime converters (DateIntervalConverter/IntervalConverter) compose over RangeConverter.. We could do something else, i.e. extract out the read/write functionality and use that from RangeConverter/DateIntervalConverter but I'm not sure it's that useful...

To your question, realistically I think very few plugins will actually need to add ranges (builtin+nodatime, do we have anything else?), so not sure it matters that much... But we could...

Reply:
Sounds good, let's leave it. Might still be useful to add the helpers for mapping though